packages/ui/src/scss/_fonts.scss

```diff
@@ -0,0 +1,27 @@
+/* Typography Foundation */
+
+$font-family:
```

Comment:
should we use it in somewhere?

Reply:
would use the following font style variables in all the sass files.